src/main/java/mate/academy/Main.java

```diff
@@ -44,12 +50,27 @@ public static void main(String[] args) {
         yesterdayMovieSession.setMovie(fastAndFurious);
         yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService
+                = (MovieSessionService) injector.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
         movieSessionService.add(yesterdayMovieSession);
 
         System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
                         fastAndFurious.getId(), LocalDate.now()));
+
+        User firstUser = new User();
+        firstUser.setEmail("first@mail.com");
+        firstUser.setPassword("firstpa$$word");
+
+        User secondUser = new User();
+        secondUser.setEmail("second@mail.com");
+        secondUser.setPassword("secondpa$$word");
+
+        UserService userService = (UserService) injector.getInstance(UserService.class);
+        userService.add(firstUser);
+        userService.add(secondUser);
+        System.out.println(userService.findByEmail(firstUser.getEmail()));
+        System.out.println(userService.findByEmail(secondUser.getEmail()));
```

Comment:
I think we need to test AuthenticationService here (register() and login())

Reply:
Many thanks for your comment! The main method was fixed!